samples/bluetooth/central_gatt_write/src/gatt_write_common.c

```diff
@@ -46,6 +56,120 @@ static void write_cmd_cb(struct bt_conn *conn, void *user_data)
 		write_len = 0U;
 		write_rate = 0U;
 		cycle_stamp = k_cycle_get_32();
+
+#if defined(CONFIG_BT_USER_PHY_UPDATE)
+		const struct bt_conn_le_phy_param phy_param[] = {
```

Comment:
Suggest to move this entire block to a new function

Reply:
Updated.